### ISSUE TITLE
Activity links

### ISSUE THREAD

**Issue by rcollina** [dotnet/docs#39397]
### Type of issue

Missing information

### Description

It would be nice to have a ballpark figure of how many links is considered "too many". Docs mention a big batch of work - see below - is it hundreds of activities? thousands? tens of thousands? etc

```csharp
void DoBatchWork(ActivityContext[] requestContexts)
{
    // Assume each context in requestContexts encodes the trace-id that was sent with a request
    using(Activity activity = s_source.StartActivity(name: "BigBatchOfWork",
                                                     kind: ActivityKind.Internal,
                                                     parentContext: default,
                                                     links: requestContexts.Select(ctx => new ActivityLink(ctx))
    {
        // do the batch of work here
    }
}
```

### Page URL

https://learn.microsoft.com/en-us/dotnet/core/diagnostics/distributed-tracing-instrumentation-walkthroughs

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/core/diagnostics/distributed-tracing-instrumentation-walkthroughs.md

### Document Version Independent Id

dc4689df-b03a-e744-dbed-f23d8defcb6e

### Article author

@tommcdon

### Metadata

* ID: 3f30f3b0-90cb-d2eb-ee08-4319ffcb7686 
* Service: **dotnet-fundamentals**

**Comment by tommcdon** [comment]
@tarekgh @noahfalk

**Comment by tarekgh** [comment]
According to the specs the recommended limit is 128. https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-limits. 

@rcollina are you interested to submit a doc PR for that?

**Comment by rcollina** [comment]
@tarekgh I'd love to! what are the steps?

**Comment by tarekgh** [comment]
@rcollina thanks for willing to help. 

Simply clone the repo https://github.com/dotnet/docs/. In the cloned copy create a branch, edit the file `docs/core/diagnostics/distributed-tracing-instrumentation-walkthroughs.md` and push your created branch. Open a PR request to push your branch to the dotnet main branch. Let me know if this is not clear so I can send more details as needed.

**Comment by BartoszKlonowski** [comment]
@rcollina Remember that you first have to fork this repository, and only after that clone your fork of this repo and push your created branch to your fork. You won't be able to push any branches to clone of this repo without write permissions.

**Comment by tarekgh** [comment]
Thanks @BartoszKlonowski. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork may be a useful link to explore.

**Comment by rcollina** [comment]
Thank you all, I'll get to it as soon as I can.
